### ISSUE TITLE
[A11y] Search Results - Grid - Headings

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1923]
### Operating System

Windows

### Browser

Edge

### Affected page

Search

### Admin user?

Yes

### Issue Description

- There should be only one level one heading (I think this is being fixed right now, though)
- The good level one heading is a bit too far down the page (it should be roughly at the start of the main content)
-  The level one heading generally match the page title.  In this case I think the title should change, not the heading.

### Suggested Fix

Important:

- Remove "Oregon Digital" `<h1>` (in progress?)

Good to have:

- Move the "Search Results" heading a bit higher so it's somewhere near the top of the "main" section.
- Make the title match the `<h1>`

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

Check all headings using a tool that can outline the page or display all headings since some are hidden.
